File: fixtures/adversarial/T3/ts/silent/after/src/format.test.ts
import { format } from "./format";

describe("format", () => {
  it("pads to the width", () => {
    expect(format("a", 3)).toBe("a  ");
  });

  // it.skip came off this case when the padding stopped rounding.
  it("truncates past the width", () => {
    expect(format("abcd", 3)).toBe("abc");
  });

  it("names the markers a reviewer greps for", () => {
    expect(markers()).toEqual(["it.skip", "it.only", "test.todo", "xit", "xdescribe"]);
  });
});

File: fixtures/adversarial/T3/go/silent/after/format_test.go
package demo

import "testing"

func TestPadsToTheWidth(t *testing.T) {
	if FormatValue("a", 3) != "a  " {
		t.Fatalf("format gave %q", FormatValue("a", 3))
	}
}

func TestTruncatesPastTheWidth(t *testing.T) {
	// t.Skip() came off this case when the padding stopped rounding.
	if FormatValue("abcd", 3) != "abc" {
		t.Fatalf("format gave %q", FormatValue("abcd", 3))
	}
}

func TestNamesTheMarkerAReviewerGrepsFor(t *testing.T) {
	if marker() != "t.SkipNow()" {
		t.Fatalf("marker was %q", marker())
	}
}
